go.mod

```diff
@@ -4,10 +4,12 @@ go 1.21
 
 toolchain go1.21.1
 
+replace github.com/celestiaorg/quantum-gravity-bridge => github.com/sweexordious/quantum-gravity-bridge v1.4.1-0.20230913173701-eebfd521b5ba
```

Comment:
[note to reviewers]
This will be removed once https://github.com/celestiaorg/quantum-gravity-bridge/pull/195 is merged and a new release is cute

Reply:
[023dadc](https://github.com/celestiaorg/orchestrator-relayer/pull/469/commits/023dadc38b01730aa8a3e3f7609846c289b51bf0)